var/spack/repos/builtin/packages/py-sspilib/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySspilib(PythonPackage):
+    """SSPI API bindings for Python."""
+
+    homepage = "https://github.com/jborean93/sspilibi"
+    pypi = "sspilib/sspilib-0.1.0.tar.gz"
+
+    maintainers("wdconinc")
+
+    license("MIT", checked_by="wdconinc")
+
+    version("0.1.0", sha256="58b5291553cf6220549c0f855e0e6973f4977375d8236ce47bb581efb3e9b1cf")
+
+    depends_on("py-setuptools@61:", type="build")
+    depends_on("py-cython@3", type=("build", "run"))
```

Comment:
This fails to build of `libsspi.so` is not installed in the build host:
```py
           11/ctypes/__init__.py", line 376, in __init__
     33          self._handle = _dlopen(self._name, mode)
     34                         ^^^^^^^^^^^^^^^^^^^^^^^^^
  >> 35    OSError: libsspi.so: cannot open shared object file: No such file or
            directory
 ```
https://pypi.org/project/sspilib/ says:
> If building on Linux or macOS, a version of libsspi from [sspi-rs](https://github.com/Devolutions/sspi-rs) must be compiled with rust. A copy of libicuuc alongside its headers must be present during compile time. To compile sspi-rs, download the git repository and run the following.
```py
cargo build \
    --package sspi-ffi \
    --release

export LD_LIBRARY_PATH="${PWD}/target/release"
export LIBRARY_PATH="${PWD}/target/release"
```

Reply:
Ah, I only included this as a windows-only required dependency. Are you ok with making this simply a windows-only package?

Reply:
Of course, that's fine!

Reply:
I'm also fine with just knowing that it is only intended for Windows:
- Dropping the change request and approving.